src/services/tree-sitter/languageParser.ts

```diff
@@ -18,12 +20,17 @@ import {
 export interface LanguageParser {
 	[key: string]: {
 		parser: Parser
-		query: Parser.Query
+		query: Query
 	}
 }
 
 async function loadLanguage(langName: string) {
-	return await Parser.Language.load(path.join(__dirname, `tree-sitter-${langName}.wasm`))
+	if (process.env.NODE_ENV === "test") {
```

Comment:
Inspired by continue.dev; allow tests to loading language syntax trees so we don't have to mock.